substrate/substrate-dump/src/dumper.ts

```diff
@@ -107,6 +114,7 @@ export class Dumper {
 
         let height = new Throttler(() => this.src().getFinalizedHeight(), 300_000)
         let chainHeight = await height.get()
+        this.prometheus().setLastBlock(this.options.lastBlock ? this.options.lastBlock : chainHeight);
```

Comment:
Metric should mean exactly one thing.

Reply:
While I agree, I still not understand purpose of this metric when the `--last-block` specified

Reply:
RPC node can be behind `--last-block`